packages/structure-viewer/src/redux.js

```diff
@@ -117,13 +117,10 @@ export const actions = {
        </queryRefinement>
       </orgPdbCompositeQuery>`
       dispatch(actions.startPdbSearch(currentGene))
-      superagent.post('http://www.rcsb.org/pdb/rest/search') // ?sortfield=Residue Count
+      return superagent.post('http://www.rcsb.org/pdb/rest/search') // ?sortfield=Residue Count
```

Comment:
Returning a promise was necessary for test code that waits until the following actions have been dispatched (previously the test would sleep for 2 seconds and assume the action had been dispatched by then).